client/src/App.js

```diff
@@ -1,18 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState } from "react";
+import { Route } from "react-router-dom";
 
-import SavedList from './Movies/SavedList';
+import SavedList from "./Movies/SavedList";
+import MovieList from "./Movies/MovieList";
+import Movie from "./Movies/Movie";
+import MovieCard from "./Movies/MovieCard";
 
-const App = () => {
-  const [savedList, setSavedList] = useState( [] );
+const App = props => {
+  const [savedList, setSavedList] = useState([]);
 
   const addToSavedList = movie => {
-    setSavedList( [...savedList, movie] );
+    setSavedList([...savedList, movie]);
   };
 
   return (
     <div>
       <SavedList list={savedList} />
-      <div>Replace this Div with your Routes</div>
+      <Route exact path="/" component={MovieList} />
+      {/* <Route path="/movies/movie/:id" render={props => <Movie {...props} addToSavedList={addToSavedList} /> } /> */} 
```

Comment:
line 20 looks accurate to me other than it's missing the `exact` prop.  Why did you change it to the component prop over the render prop?